README.md

```diff
@@ -57,6 +57,7 @@ Sentry instruments your application with observability, such as error tracking a
 
 - [Setting up Sentry Logs for Next.js](https://www.youtube.com/watch?v=m3zuWITW-yI) - Walkthrough of setting up Sentry logging to debug Next.js applications.
 - [Setting up Sentry Logs for Laravel](https://www.youtube.com/watch?v=isyAwH9t68M&t=1s) - Walkthrough of setting up Sentry logging to debug Laravel applications.
+- [Improving Browser Tracing Step by Step](https://www.youtube.com/watch?v=iUxyRV99AwY) - A video presenting the newest Browser Tracing features in Sentry's SDK as of Sep 29, 2025
```

Comment:
Perhaps explicitly call out the ability to ignore spans? That was the topic that made me want to share this update with people.